resource_estimator/src/estimates/modeling/physical_qubit.rs

```diff
@@ -13,6 +13,64 @@ use super::super::{
 };
 use serde::{de::Error, Deserialize, Serialize};
 
+pub trait TPhysicalQubit {
+    fn instruction_set(&self) -> PhysicalInstructionSet;
+    fn t_gate_error_rate(&self) -> f64;
```

Comment:
Just to confirm that we expect all new qubits to support those properties, and we do not need more formulas. 

For example, currently we use those error rates for t-factories construction, e.g. in custom formulas. Should we consider a wider (flexible?) set of custom formulas where we would consume not 3 but more parameters in polynomials?

Reply:
Possibly yes, for now I was just extracting the interface based on the current types and made small adjustments on the ErrorCorrection trait.

Reply:
All these properties are part of the implementation of the error correction, there is no more trait for physical qubits.